index.html

```diff
@@ -242,7 +242,7 @@ <h1 class="title">About</h1>
           </p>
           <div class="aboutbtn">
             <button>
-              Get my CV <i class="fa-solid fa-arrow-right-to-bracket"></i>
+              Get my CV <i class="fa-solid fa-arrow-right-to-bracket" alt="Arrow-icon"></i>
             </button>
```

Comment:
- I think your `Get My CV` button is currently failing on hover when tested on [aditus tool](https://www.aditus.io/button-contrast-checker/). Can we fix it?

Here is the result of the color contrast test 👇🏽 

![hovf](https://user-images.githubusercontent.com/99246359/207365768-8720e67e-edf7-427e-95f3-d0db9bc4eb57.PNG)

- I believe you can fix this either by using a whiter shade for your text color, or a darker shade for your background. Anyone you choose should work. 🤗

⚠️ Kindly note that this should be implemented only on the `hover` style. Also, it's important that you remove the `alt` attribute added to the button's icon as well. 🍒